distributed/cli/dask_scheduler.py

```diff
@@ -64,6 +67,9 @@ def del_pid_file():
         limit = max(soft, hard // 2)
         resource.setrlimit(resource.RLIMIT_NOFILE, (limit, hard))
 
+    if interface and not host:
```

Comment:
We should probably disallow passing both host and interface, instead of sliently ignoring one of them.